repos/fdbt-site/src/layout/Layout.tsx

```diff
@@ -71,6 +82,53 @@ export const BaseLayout = ({
     );
 };
 
+export const MyFaresLayout = ({
```

Comment:
What's all this? This looks like duplicated code?

Reply:
Yeah I think its a hangover from hack day branch.